frontend/src/components/landing-page/landing-page.tsx

```diff
@@ -53,10 +56,16 @@ export const LandingPage = (): React.ReactElement => {
         <div className={styles.secondContainer}>
           <div className={styles.secondContainerHeader}>New Cars</div>
           <div className={styles.secondContainerCards}>
-            <NewCarCard />
-            <NewCarCard />
-            <NewCarCard />
-            <NewCarCard />
+            {!cars
+              ? null
+              : cars?.map((car) => (
+                  <NewCarCard
+                    type={'complectation'}
+                    isLiked={false}
+                    car={car}
+                    key={car.id}
+                  />
+                ))}
```

Comment:
You don't need to use the ternary operator here. Use the `&&` one - `cars && cars.map(...)`